docs/packaging/monitoring.yml.md

```diff
@@ -17,6 +17,26 @@ Checking for new releases is done by mapping the Solus package to an [Anitya](ht
 
 Checking for security advisories ([CVEs](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures)), is done by mapping the Solus package to a _Common Platform Enumeration Name_ ([CPE](https://nvd.nist.gov/products/cpe)) from the National Vulnerability Database.
 
+## Adding monitoring.yml to an existing package
+
+To add a monitoring.yml file to an existing package you can use the following `go-task` commands to add a template file, starting from within the Packages repository:
+
+```bash
+go-task add-monitoring
+```
+
+Which will create a file with the following contents:
+
+```text
```

Comment:
The example of a monitoring.yml file uses ```yaml to get yaml highlighting.
Here, the language is specified as text. Is it intentional to avoid syntax highlighting here? Personally I think it would be better to have it for readability and to make this more consistent with the other yaml code on the page.

Reply:
nah, it should be yaml